client-app/ui-kit/components/atoms/input-details/vc-input-details.vue

```diff
@@ -53,7 +53,7 @@ withDefaults(defineProps<IProps>(), {
   $error: "";
   $singleLine: "";
 
-  @apply flex mt-0.5 gap-2 min-h-[0.875rem] text-xs;
+  @apply flex mt-1 gap-2 min-h-[0.875rem] text-xs;
```

Comment:
according to the design there is 4 px margin between input and details (for inputs, checkboxes, textareas etc.)